### ISSUE TITLE
v4: Shadow is not working on iOS

### ISSUE THREAD

**Issue by ghmendonca** [marklawlor/nativewind#632]
**Describe the bug**
The `shadow` class is not working anymore, it's not applying any style to the component. I tested on web and it works, but I didn't test on Android.

**To Reproduce**
1. Create a new `<View />` and add some width and height.
2. Add `className="shadow"`
4. See that the shadow is not there

**Expected behavior**
A shadow should appear. This was working on v2, so it's definitely something related to v4. Didn't test on v3.

**Comment by marklawlor** [comment]
iOS shadows are finicky, even with inline styles. Its a tricky issue because there are multiple reasons why shadow might not work, e.g missing background-colors, view overflows, zIndexing, etc.

Can you first confirm that applying the shadow via inline styles works? If the inline styles work but not the NativeWind ones can you please provide a full reproduction and I can look into it further

**Comment by ghmendonca** [comment]
@marklawlor thank you for the answer. Yes, I will confirm and in case the inline style works, I will create a new repo to confirm the issue

**Comment by ghmendonca** [comment]
@marklawlor ok, I just confirmed that the issue is that the `shadow` class is not setting the `shadowOpacity` style and that's why is not working on iOS. Right now, the shadow class outputs:

```
{
  elevation: 2,
  shadowColor: "rgba(0, 0, 0, 0.10196078568696976)",
  shadowOffset: {
    height: 1,
    width: 0,
  },
  shadowRadius: 2,
}
```

And in order to work on iOS, the `shadowOpacity` style should be defined, so using the style below works:

```
{
  elevation: 2,
  shadowColor: "rgba(0, 0, 0, 0.10196078568696976)",
  shadowOffset: {
    height: 1,
    width: 0,
  },
  shadowRadius: 2,
  shadowOpacity: 0.1
}
```

Here is the repo so you can try yourself: https://github.com/ghmendonca/nativewind-shadow

I would be glad to help you fixing this, I was looking at the code but I'm not sure if I understand exactly how everything works, I don't know where you convert the classes into styles, so if you could point me what needs to be done in order to fix this, I can open a PR for you

**Comment by enchorb** [comment]
+1

**Comment by marklawlor** [comment]
Fixed in `4.0.14`
